SoftLayer/fixtures/SoftLayer_Virtual_Guest.py

```diff
@@ -920,3 +920,23 @@
     "createDate": "2021-03-22T13:15:31-06:00",
     "id": 1234567
 }
+
+getBrowserConsoleAccessLogs = [
+    {
+        "createDate": "2022-03-07T05:32:23-06:00",
+        "eventType": "CONNECTED",
+        "id": 509636,
+        "message": "User connected",
+        "sourceIp": "44.200.9.0",
+        "sourcePort": 47097,
+        "username": "307608_sebastian.chaparro@ibm.com"
```

Comment:
Dont put real emails/usernames in the fixtures, compliance doens't like that.
```suggestion
        "username": "testuser@ibm.com"
```